KerbalStuff/blueprints/api.py

```diff
@@ -676,5 +678,6 @@ def update_mod(mod_id):
     db.commit()
     mod.default_version_id = version.id
     db.commit()
-    notify_ckan.delay(mod_id, 'update')
-    return { 'url': url_for("mods.mod", mod_id=mod.id, mod_name=mod.name), "id": version.id  }
+    if mod.ckan:
+        notify_ckan.delay(mod_id, 'update')
```

Comment:
I'm not sure if it's going to work. If a mod didn't enable CKAN on creation and then enabled it, the update event should fail because it will be called for a non-existent mod.

Reply:
Turns out it's not possible to enable CKAN for a mod without using the web UI, something we should definitely look at in the future. Marking resolved because the condition specified above will never happen, because the UI is going to call `send_to_ckan` before the API notifies CKAN.